laravel/app/Console/Commands/PushSchedule.php

```diff
@@ -47,15 +68,125 @@ public function __construct()
-     * @return Array daily assignments to be output to the Google Sheet
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        // connect to the api and our sheet
+        $client = self::getClient();
+        $service = new Google_Service_Sheets($client);
+
+        // TODO: environment variable
+        $spreadsheetId = '1npNBs_j6BvmZO29GHlEJ-mROGhtBEqM7_KNKdAnNLxY';
+
+        $tomorrow = date('Y-m-d', strtotime('+1 day'));
+        $title = $tomorrow;
+
+        // create new sheet for today
+        $newSheet = new Google_Service_Sheets_Spreadsheet([
+            'properties' => [
+                'title' => $title,
+                'index' => 0,
+            ],
+        ]);
+        $newSheet = $service->spreadsheets->create($newSheet, ['fields' => 'spreadsheetId']);
+        Log::info("Created spreadsheet: \"$title\" with ID: $newSheet->spreadsheetId");
+
+        // get the array that contains all assigments for day + 1.
+        $all_assns = self::getAssignments($tomorrow);
+
+        // relative path for assignment sheet
+        $assignmentCSVPath = "../downloads/assignment$tomorrow.csv";
+        $mode = file_exists($assignmentCSVPath) ? 'w' : 'c';
+        $fp = fopen($assignmentCSVPath, $mode);
+
+        // print header for sheet
+        fputcsv($fp, PushSchedule::$SPREADSHEET_COLUMN_NAMES);
+        // print all assignments to path
+        foreach ($all_assns as $assignemnts) {
+            fputcsv($fp, $assignemnts);
+        }
+
+        fclose($fp);
+
+        $range = "'$title'!A1:ZZZ1000";
+
+        // get the values from the assignment file and save them to an array
+        $file = fopen($assignmentCSVPath, 'r');
+        $csv = [];
+        while (($line = fgetcsv($file)) !== false) {
+            // $line is an array of the csv elements
+            $csv[] = $line;
+        }
+        fclose($file);
+
+        // create the correct update to send back to google sheets to fill the sheet with the array created above
+        $body = new Google_Service_Sheets_ValueRange(['values' => $csv]);
+        $params = ['valueInputOption' => 'USER_ENTERED'];
+        $result = $service->spreadsheets_values->update($spreadsheetId, $range, $body, $params);
+
+        // remove the oldest spreadsheet if there are over MAX_SPREADSHEETS spreadsheets
+        $response = $service->spreadsheets->get($spreadsheetId);
+        if (count($response) > PushSchedule::$MAX_SPREADSHEETS) {
+            $lastEntry = $response[count($response) - 1];
+            $lastSheetId = $lastEntry['properties']['sheetId'];
+            $delete = new Google_Service_Sheets_BatchUpdateSpreadsheetRequest([
+                'requests' => [
+                    'deleteSheet' => [
+                        'sheetId' => $lastSheetId,
+                    ],
+                ],
+            ]);
+            $service->spreadsheets->batchUpdate($spreadsheetId, $delete);
+        }
+
+        return 0;
```

Comment:
Why are we returning 0?

Reply:
new versions of Laravel default to this.